tests/suites/test_suite_psa_its.function

```diff
@@ -190,14 +199,15 @@ void get_at( int uid_arg, data_t *data,
     size_t length = length_arg >= 0 ? length_arg : 0;
     unsigned char *trailer;
     size_t i;
+    size_t ret_len = 0;
 
     ASSERT_ALLOC( buffer, length + 16 );
     trailer = buffer + length;
     memset( trailer, '-', 16 );
 
     PSA_ASSERT( psa_its_set_wrap( uid, data->len, data->x, 0 ) );
 
-    status = psa_its_get( uid, offset, length_arg, buffer );
+    status = psa_its_get( uid, offset, length_arg, buffer, &ret_len );
```

Comment:
We should probably assert that `ret_len` is equal to `length` when `status == PSA_SUCCESS`

Reply:
Done